etc/profile-a-l/cointop.profile

```diff
@@ -0,0 +1,56 @@
+# Firejail profile for cointop
+# Description: cointop is a fast and lightweight interactive terminal based UI application for tracking and monitoring cryptocurrency coin stats in real-time. 
```

Comment:
Please avoid marketing speak in profile descriptions.

I'd say to just make them short and to the point, similarly to the ones from
distro packages.  For example, this is what is used in the AUR:

```console
$ yay -Sii cointop | grep '^Description'
Description     : An interactive terminal based UI application for tracking cryptocurrencies
```

I think that it could be further simplified to the following:

> Description: TUI for tracking cryptocurrency stats